stable/kubewatch/values.yaml

```diff
@@ -6,6 +6,19 @@ slack:
   # and invite the bot to your channel using: /join @botname
   token: "XXXX"
 
+hipchat:
```

Comment:
wouldnt be better if this values were commented out and the chat checked if they existed rather than adding them as blank to the values.yaml?

Reply:
yes, I will make the changes